src/adventures-photography/_posts/2017-09-24-helvellyn.md

```diff
@@ -0,0 +1,80 @@
+excerpt: "Spending a night in a bivvy bag on the summit of Helvellyn"
+thumb: 2017-09-web-DSCF5918
+banner: 2017-09-web-DSCF5918
+tags: bikepacking microadventure cumbria
+date: 2017-09-24 18:10
+---
+
+After spending the previous week in the Lake District National Park I decided to end the trip with something a little different. Armed with a basic overnight kit I left Glenridding at just gone 16:00 — when most people were heading back down — for the summit of Helvellyn, a 950m mountain which features a large and rocky summit with a small wind break on the summit: and it was this wind break I had my eye on to provide shelter for the night. 
+
+I headed up past the youth hostel towards Red Tarn where others were already setting up their tents for a night under the stars. At 820m I still had over 100m of ascent left before I’d reach my home for the night. I was coming up this route simply to cut the weight I had to haul with me: I’d need water for my dehydrated beef stew dinner, as well as to drink once on the summit where there is no water. I’d have to carry everything I need, so I stopped here to fill my Platypus and Nalgene bottles with 3 litres of Red Tarns finest mineral water, to which I quickly added a few chlorine tablets for safety. 
+
+{% figure %}
+    {% img src: 2017-09-web-DSCF5737-HDR, alt: Looking down towards Red Tarn just a few meters from the Helvellyn summit shelter. %}
+    {% figcaption %}Looking down towards Red Tarn just a few meters from the Helvellyn summit shelter{% endfigcaption %}
+{% endfigure %}
+
+Continuing my ascent I now headed along the path back towards the hole in the wall, before doubling back along the start of Striding Edge, a knife-edge ridge approach onto the summit of Helvellyn with steep sides which could result in severe consequence in the event of a fall. There was only a gentle breeze so I was happy to commit to the 1.5km ridge having previously completed it, despite my heavy pack.
+
+Taking it steady I made my way along the ridge carefully navigating the features as to not slip or trip — something which could be fatal at the best of times here. The entire ridge was just as I remember it from when I last completed it a couple of years back. 
+
+Reaching the summit I dropped my bag on the bench in the wind break (simply marked as “shelter” on OS maps at grid NY342150) before looking around at the evening light falling over the hills all around me. The views were absolutely stunning; the next couple of hours were spent running around with the camera. 
+
+{% figure %}
+    {% img src: 2017-09-web-DSCF5740-Pano, alt: Evening light settling over Swirrels Edge (my intended descent) and Catstye Cam. %}
+    {% figcaption %}Evening light settling over Swirrels Edge (my intended descent) and Catstye Cam.{% endfigcaption %}
+{% endfigure %}
+
+As the sunset I spread my bivvy bag out on a bench and settled down to make some dinner and coffee before getting my head down the the night. 
```

Comment:
> As the sunset

Should this be:

> As the sun set

Or,

> At the sunset

Reply:
Thanks! All fixes, that was super helpful :)